data/utils.ts

```diff
@@ -17,14 +17,14 @@ export const BUILDOZER_BIN = path.join(process.cwd(), 'bin', 'buildozer')
 
 export interface Metadata {
   homepage?: string
-  maintainers: Array<{
```

Comment:
nit: we could consider doing the opposite. We already have a schema validator in the bcr that checks all module metadata is conformant.

(and this Metadata schema ought to be imported from that JSON schema so we know the two repos agree on required fields, etc)

WDYT?

Reply:
sure -- I just have no idea how to do any of that :)

I suggest we submit this PR first to make sure we have an updated website build, and then worry about finding engineering time for your suggestion.

Reply:
I'm also more in favor of having a JSON schema validation in BCR, which can serve as a source of truth, as that also would make implementation of alternative registry UIs easier.

But yeah, let's go ahead and merge this for now, as this seems to be something that happens somewhat regularly now.

Reply:
Opened up an issue for it here: https://github.com/bazelbuild/bazel-central-registry/issues/1365